Source/Renderer/ShaderSource.js

```diff
@@ -330,28 +332,33 @@ define([
 
     ShaderSource.createPickVertexShaderSource = function(vertexShaderSource) {
         var renamedVS = ShaderSource.replaceMain(vertexShaderSource, 'czm_old_main');
-        var pickMain = 'attribute vec4 pickColor; \n' +
-            'varying vec4 czm_pickColor; \n' +
-            'void main() \n' +
-            '{ \n' +
-            '    czm_old_main(); \n' +
-            '    czm_pickColor = pickColor; \n' +
-            '}';
-
+        var pickMain = 
+//language=glsl            
+'\
+attribute vec4 pickColor;\n\
+varying vec4 czm_pickColor;\n\
+void main()\n\
+{\n\
+    czm_old_main();\n\
+    czm_pickColor = pickColor;\n\
+}';
         return renamedVS + '\n' + pickMain;
     };
 
     ShaderSource.createPickFragmentShaderSource = function(fragmentShaderSource, pickColorQualifier) {
         var renamedFS = ShaderSource.replaceMain(fragmentShaderSource, 'czm_old_main');
-        var pickMain = pickColorQualifier + ' vec4 czm_pickColor; \n' +
-            'void main() \n' +
-            '{ \n' +
-            '    czm_old_main(); \n' +
-            '    if (gl_FragColor.a == 0.0) { \n' +
-            '       discard; \n' +
-            '    } \n' +
-            '    gl_FragColor = czm_pickColor; \n' +
-            '}';
+        var pickMain = pickColorQualifier + 
+//language=glsl                       
+' \
```

Comment:
I think for the sake of clarity they should be on the same line, otherwise it looks like `vec4 czm_pickColor` is the start of a line when it isn't.

```
        var pickMain = pickColorQualifier + 'vec4 czm_pickColor;\n' +
//language=glsl                       
' \
```